gpkit/tests/t_model.py

```diff
@@ -423,35 +436,35 @@ def test_trivial_sp2(self):
     def test_sp_initial_guess_sub(self):
         x = Variable("x")
         y = Variable("y")
-        x0 = 1
+        x0 = 2
         y0 = 2
         with SignomialsEnabled():
-            constraints = [y + x >= 2, y <= x]
+            constraints = [y + x >= 4, y <= x]
         objective = x
         m = Model(objective, constraints)
         try:
             sol = m.localsolve(x0={x: x0, y: y0}, verbosity=0,
                                solver=self.solver)
         except TypeError:
             self.fail("Call to local solve with only variables failed")
-        self.assertAlmostEqual(sol(x), 1, self.ndig)
-        self.assertAlmostEqual(sol["cost"], 1, self.ndig)
+        self.assertAlmostEqual(sol(x), 2, self.ndig)
+        self.assertAlmostEqual(sol["cost"], 2, self.ndig)
 
         try:
             sol = m.localsolve(x0={"x": x0, "y": y0}, verbosity=0,
                                solver=self.solver)
         except TypeError:
             self.fail("Call to local solve with only variable strings failed")
-        self.assertAlmostEqual(sol("x"), 1, self.ndig)
-        self.assertAlmostEqual(sol["cost"], 1, self.ndig)
+        self.assertAlmostEqual(sol("x"), 2, self.ndig)
+        self.assertAlmostEqual(sol["cost"], 2, self.ndig)
 
         try:
             sol = m.localsolve(x0={"x": x0, y: y0}, verbosity=0,
                                solver=self.solver)
         except TypeError:
             self.fail("Call to local solve with a mix of variable strings "
                       "and variables failed")
-        self.assertAlmostEqual(sol["cost"], 1, self.ndig)
```

Comment:
I wanted to make sure that the initial guess was different from the default guess :)